pages/about.html

```diff
@@ -100,28 +100,79 @@
             Hevons Farm is a subsidiary of Hevons Synergy, a company specialized
             in general contracting, Resource management, Human resources and
             Pre/Post production processing.
           </p>
           <p class="mt-5">
             Please contact us for your questions and enquiries on
             <br />
-            <a href="mailto:makojidavid@gmail.com" class="text-dark">support@hevons.com</a>
+            <a href="mailto:makojidavid@gmail.com" class="text-dark"
+              >support@hevons.com</a
+            >
           </p>
         </div>
       </section>
+      <section class="bg-white py-4">
+        <h1 class="h1 text-uppercase text-center mt-5 fw-bold">Our Values</h1>
+        <div class="about-line"></div>
+        <p class="text-center text-dark mt-3 col-sm-10 col-md-8 mx-auto">
+          We are a company built on solid cultural orientatons and believes. Our
+          goal is to ensure the availability, affordability of both raw and
+          processed/market ready food items.
+        </p>
+        <div class="shadow-lg col-6 mx-auto">
+          <img src="../img/logo.png" class="w-50 h-50 logo-img" alt="logo" />
+        </div>
+      </section>
+      <section class="bg-white py-5">
+        <h1 class="h1 text-uppercase text-center mt-5 fw-bold">
+          Our Locations
+        </h1>
+        <div class="about-line"></div>
+        <div class="d-flex flex-wrap justify-content-center mt-5 col-12">
+          <div class="position-relative col-10 mx-auto col-md-5 col-lg-5 m-1 location">
+            <div class="location-txt position-absolute">
+              <p class="text-white text-center">Keffi</p>
+              <p class="text-white text-center">Keffi Cereals Farm</p>
+            </div>
+          </div>
+          <div class="position-relative col-10 mx-auto col-md-5 col-lg-5 m-1 location">
+            <div class="location-txt position-absolute">
+              <p class="text-white text-center">Keffi</p>
+              <p class="text-white text-center">Keffi Cereals Farm</p>
+            </div>
+          </div>
+        </div>
+      </section>
     </main>
+    <!-- Partners Section -->
+    <section
+      class="bg-dark d-flex flex-column d-lg-none gap-4 col-12 py-3 py-5 partners"
+    >
+      <h3 class="h3 text-center">Our Partners</h3>
+      <div class="line"></div>
+      <div class="d-flex justify-content-evenly col-12">
+        <img src="../img/bank.png" alt="bankIcon" />
+        <img src="../img/insurance.png" alt="insuranceIcon" />
+        <img src="../img/google.webp" alt="googleIcon" />
+      </div>
+    </section>
+    <footer class="bg-light bg-lg-dark text-dark d-flex gap-4 col-12 py-3 px-5">
+      <div class="col-5 py-3">
+        <img src="../img/logo.png" class="w-25" alt="logo" />
+        Hevons Synergy
+      </div>
+      <div class="col-6 px-2 py-3">&copy; 2022, Some rights reserved</div>
+    </footer>
```

Comment:
- [ ] In the desktop version of your about page, the footer should be dark. Kindly correct this issue 💯 
- Your design:
![Capture2](https://user-images.githubusercontent.com/85113567/181272000-810370e8-1f4e-45c8-8fef-f1610b04115b.PNG)
- Required design:
![Capture3](https://user-images.githubusercontent.com/85113567/181272037-db469c6b-a920-49c4-ab36-675c7080b092.PNG)

Reply:
Fixed